### PR TITLE
Update script to build and upload MiniBench artifacts

### PR DESCRIPTION
**Pull request by kirklandsign**

### CODE REVIEW

File: build/build_android_llm_demo.sh [hunk 1]
```diff
@@ -91,12 +91,18 @@ build_aar() {
   popd
 }
 
-build_android_llm_demo_app() {
+build_android_demo_apps() {
```
Comment (guangy10) [inline build/build_android_llm_demo.sh:94]: Should we rename the script and steps in android-perf as well?

**Review by guangy10 — APPROVED** [review]
Changes look good to me

**Comment by guangy10** [comment]
I think you will need https://github.com/pytorch/executorch/pull/5098 in order to run on S22

File: build/build_android_llm_demo.sh (hunk 1 above)
Comment (kirklandsign) [reply build/build_android_llm_demo.sh:94]: Yes! Let's do that later. It's really used in different places

**Comment by kirklandsign** [comment]
We can use this one later with https://github.com/pytorch/executorch/pull/5094 on generic models 😸

**Review by huydhn — APPROVED** [review]
LGTM! After this PR and once I figure out how to get the results from the device, let's sync on what we need to change on the test spec to work with minibench

**Comment by guangy10** [comment]
@kirklandsign This PR is merged. I'm wondering how it's tested? I thought you will wait for https://github.com/pytorch/executorch/pull/5098 then test the minibench in the android-perf using this PR.

**Comment by guangy10** [comment]
> @kirklandsign This PR is merged. I'm wondering how it's tested? I thought you will wait for #5098 then test the minibench in the android-perf using this PR.

https://github.com/pytorch/executorch/pull/5098 is merged. @kirklandsign can you please verify if the app works with QNN models?

**Comment by kirklandsign** [comment]
> > @kirklandsign This PR is merged. I'm wondering how it's tested? I thought you will wait for #5098 then test the minibench in the android-perf using this PR.
> 
> #5098 is merged. @kirklandsign can you please verify if the app works with QNN models?

Oh let me pull in QNN deps and I will do a verification. One moment

**Pull request merged**